cmd/cstor-pool-mgmt/controller/pool-controller/handler.go

```diff
@@ -242,13 +250,13 @@ func (c *CStorPoolController) importPool(cStorPoolGot *apis.CStorPool, cachefile
 	if err == nil {
 		err = pool.SetCachefile(cStorPoolGot)
 		if err != nil {
-			common.IsImported <- false
+			common.IsImported = false
```

Comment:
Mutex unlock should be done after this step.

Reply:
after execution of this command, we are unlocking the mux. Incase of unlocking before, we are returning back the function